src/rgw/rgw_rest_s3.cc

```diff
@@ -1687,7 +1687,9 @@ int RGWPostObj_ObjStore_S3::get_tags()
     RGWObjTagging_S3 *tagging;
 
     tagging = static_cast<RGWObjTagging_S3 *>(parser.find_first("Tagging"));
-    obj_tags_s3 = static_cast<RGWObjTagSet_S3 *>(tagging->find_first("TagSet"));
+    if(tagging){
```

Comment:
@amitkumar50 can you add some spaces to this `if` statement? e.g., `if (tagging) {`